### DISCUSSION TITLE
Gledopto GL-G-001P Firmware (Garden Light)

### DISCUSSION THREAD

**Discussion by iculookn** [Koenkk/zigbee2mqtt#12841] [Devices]
Hello, I am new and just got one of my GLEDOPTO garden lights added to test. I spoke to Gledopto and they provided an updated firmware. I have NOT tried it yet, as I am trying to learn how to manually load from a local file.

Not sure if it helps, but here is the file.
[GL-G-001P_V11076801_OTAV12_20211028.zip](https://github.com/Koenkk/zigbee2mqtt/files/8917163/GL-G-001P_V11076801_OTAV12_20211028.zip)

This is the link AX for the light
[https://www.aliexpress.com/item/1005001431521665.html?spm=a2g0o.order_list.0.0.21ef18025DGJXP](url)

**Comment by iculookn** [comment]
![image](https://user-images.githubusercontent.com/8793442/174031129-916737cd-aff3-40c8-8ea5-4f68124e7bba.png)

**Comment by Koenkk** [comment]
Added the OTA, the update should be available now when you check for an OTA update via the OTA tab.
